### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by Wineshuga**
### Description

In this branch, we carried out the following:

- Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram below:
clinic_diagram
- Included join table from many-to-many relationships.
- Added the FK indexes.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Wineshuga  🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.


- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted.  
- Kindly mkae sure that the `Key Features` and `Future Features` section includes 1-3 potential features to be added.  
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. 

For example:
image
image
image

- The License suggestion should be removed. 
image

- There should not be any blank sections that do not have content. Kindly document all your sections and update the project title
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Wineshuga** [comment]
Thank you @KDlamini for pointing this out to us. We will apply your suggestions.

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+-- create indexes
+
+CREATE INDEX patient_id_idx ON medical_histories(patient_id ASC);
+
+CREATE INDEX invoice_id_idx ON invoice_items(invoice_id ASC);
+
+CREATE INDEX treatment_id_idx ON invoice_items(treatment_id ASC);
+
+CREATE INDEX
+    medical_history_id_idx ON invoices(medical_history_id ASC);
```
Comment (DeliceLydia) [inline schema_based_on_diagram.sql]: - [ ] Please create `index` to 2 foreign keys from the join table `admission`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Created all required tables
- All tables have a PRIMARY KEY set
- All team members participated


## Required Changes ♻️
_Check the comments under the review._




### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**